public/client.js

```diff
@@ -194,7 +233,7 @@ function stopPlayer() {
 //change animation based on the direction of the player
 function changeAnimation(direction) {
     //only change animation when direction changes
-    if (player.direction !== direction) {
+    if (player.direction !== undefined && player.direction !== direction) {
```

Comment:
I think I changed this to test smth and forgot to change it back, this part: player.direction !== undefined